setup.py

```diff
@@ -60,10 +60,11 @@
     include_package_data=True,
     install_requires=[
         "importlib-metadata ; python_version<'3.8'",
-        "eth-ape>=0.1.0b2",
+        "eth-ape>=0.1.0b3",
         "eth-account>=0.5.6,<0.6.0",
         "eth-typing>=2.2.2",
         "trezor>=0.12.2",
+        "pydantic>=1.9.0",
```

Comment:
I don't think we should add pydantic just to handle signatures

Reply:
Gotcha, I'll pull it out. I was playing around with it to learn a little bit and thought it could be useful here.

Reply:
No problems, it's overkill here tho